docs/userstories/add_image.md

```diff
@@ -0,0 +1,21 @@
+```gherkin
```

Comment:
can we change the file name to something more specific than `add_image.md`?

Reply:
maybe move the template and the md file to a sub directory with a more specific name?

Reply:
I added a more specific name, I think for sub directories -- it's not clear to me what type of organization we want, I figured it would become clearer the more stories we add, for now I was thinking just dumping everything in the user-story directory would be sufficient. But I'm open to suggestions.